### PR TITLE
fix broken links and add some info about sig-robotics

### LINKED ISSUE #136: sig-robotics community links are broken.
SIG information described in https://github.com/kubeedge/community/tree/master/sig-robotics, there are some broken links that are not valid site provided.

e.g

- Meeting note and agenda links back to https://github.com/kubeedge/community/tree/master/sig-robotics
- Meeting calendar links back to https://github.com/kubeedge/community/tree/master/sig-robotics
- Is there `#sig-robotics` channel in KubeEdge slack available? https://kubeedge.slack.com/archives/CDXVBS085/p1676354247383449

Best,

### PR DESCRIPTION
**Pull request by JoeyHwong-gk**
fix #136 

Signed-off-by: JoeyHwong-gk <joeyhwong@gknow.cn>

### CODE REVIEW

**Comment by JoeyHwong-gk** [comment]
CC: @fisherxu @kevin-wangzefeng

File: sig-robotics/README.md [hunk 1]
```diff
@@ -7,11 +7,29 @@ The [charter](charter.md) defines the scope and governance of the Robotics Speci
...
+The Chairs of the SIG run operations and processes governing the SIG.
+
+*TODO*
+
+### Technical Leads
+
+The Technical Leads of the SIG establish new subprojects, decommission existing subprojects, and resolve cross-subproject technical issues and decisions.
+
+- Zhengguang Ou (@ugvddm), Huawei
+- Fisher Xu (@fisherxu), Huawei
```
Comment (fisherxu) [inline sig-robotics/README.md:33]: Can remove me at here :) But I'll still help the development of the SIG.  @ugvddm @JoeyHwong-gk

Comment (fisherxu) [reply sig-robotics/README.md:33]: Others looks good.

Comment (JoeyHwong-gk) [reply sig-robotics/README.md]: thanks for comments, fixed

File: sig-robotics/README.md [hunk 2]
```diff
@@ -7,11 +7,29 @@ The [charter](charter.md) defines the scope and governance of the Robotics Speci
...
- * [Meeting notes and Agenda]().
- * [Meeting Calendar]() | [Subscribe]()
+* Regular SIG Meeting: [Tuesday at 10:00 UTC+8](https://zoom.us/my/kubeedge) (biweekly, starts on 22th, June 2021). [Convert to your timezone](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=GMT%2B8&).
+ * [Meeting notes and Agenda](https://docs.google.com/document/d/1i-DTZTsj2GjKHlocpkyzfEO--zYvx7sZ4bEFoJCy4YM/edit#).
+ * [Meeting Calendar](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MGRnMGJzdjhmY3E2aDhhOTZlOXV0M211dXBfMjAyMzAyMDdUMDIwMDAwWiBodWFuZzY4OTRAbQ)
 
 ## Contact
-- Slack: [#sig-robotics]()
+
+- [#sig-Robotics slack channel](https://kubeedge.slack.com)
```
Comment (fisherxu) [inline sig-robotics/README.md:16]:
```suggestion
- [#sig-Robotics slack channel](https://join.slack.com/t/kubeedge/shared_invite/enQtNjc0MTg2NTg2MTk0LWJmOTBmOGRkZWNhMTVkNGU1ZjkwNDY4MTY4YTAwNDAyMjRkMjdlMjIzYmMxODY1NGZjYzc4MWM5YmIxZjU1ZDI)
```

It's better to use the invitation link here :) Others LGTM. cc @JoeyHwong-gk

Comment (JoeyHwong-gk) [reply sig-robotics/README.md]: perfect, thx. /ping @fisherxu

File: sig-robotics/README.md [hunk 3]
```diff
@@ -7,11 +7,29 @@ The [charter](charter.md) defines the scope and governance of the Robotics Speci
...
+*TODO*
+
+### Technical Leads
+
+The Technical Leads of the SIG establish new subprojects, decommission existing subprojects, and resolve cross-subproject technical issues and decisions.
+
+- Zhengguang Ou (@ugvddm), Huawei
+- JoeyHwong (@JoeyHwong-gk)
+
+> The co-chair is still open for recruitment. If any individual or organization is willing to participate in our community work, please send an email to [kubeedge](kubeedge@googlegroups.com).
```
Comment (fujitatomoya) [inline sig-robotics/README.md:35]: I am not sure how much I can commit this, but would like to join if I am eligible to this participation.

**Review by fujitatomoya — COMMENTED** [review]
@JoeyHwong-gk thanks for addressing this, looks good to me.

File: sig-robotics/README.md (hunk 3 above)
Comment (JoeyHwong-gk) [reply sig-robotics/README.md:35]: Awesome, thanks for reaching out! We're absolutely thrilled that you're interested in joining our community, and we're excited to see the unique perspectives and contributions you'll bring to the table.
We think your background is a perfect fit for our community, and we can't wait to have you on board. 

Regarding the `co-chair` selection, we'll reach out to the KubeEdge TSC (@kevin-wangzefeng @fisherxu) to create a document outlining the guidelines and requirements for the role, and we'll make sure to publish it in the community to keep everyone in the loop.

Thanks again for your enthusiasm and interest in KubeEdge - we're lucky to have you!

File: sig-robotics/README.md [hunk 4]
```diff
@@ -7,11 +7,34 @@ The [charter](charter.md) defines the scope and governance of the Robotics Speci
...
+
+- Zhengguang Ou (@ugvddm)
+- JoeyHwong (@JoeyHwong-gk)
+
+### Co-Chairs
+
+> The co-chairs will be elected on an annual basis in November for a one year term. The co-chairs will be responsible for making day-to-day decisions and overseeing Sig-Robotics’ growth.
+> If any individual or organization is willing to participate in our community work, please send an email to [kubeedge](kubeedge@googlegroups.com).
+
+- Fujita Tomoya (@@fujitatomoya)
```
Comment (fujitatomoya) [inline sig-robotics/README.md:40]: typo?

```suggestion
- Fujita Tomoya (@fujitatomoya)
```

**Review by fujitatomoya — CHANGES_REQUESTED** [review]
minor comment, could you check just in case?

**Comment by JoeyHwong-gk** [comment]
> @JoeyHwong-gk The DCO check is still failing, could you fix it?

fixed. Thank you for your kind reminder, and I apologize for the oversight.

File: sig-robotics/README.md [hunk 5]
```diff
@@ -7,11 +7,34 @@ The [charter](charter.md) defines the scope and governance of the Robotics Speci
...
- * [Meeting notes and Agenda]().
- * [Meeting Calendar]() | [Subscribe]()
+* Regular SIG Meeting: [Wednesday at 10:00 UTC+8](https://zoom.us/my/kubeedge) (biweekly, starts on 22th, June 2021). [Convert to your timezone](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=GMT%2B8&).
+ * [Meeting notes and Agenda](https://docs.google.com/document/d/1i-DTZTsj2GjKHlocpkyzfEO--zYvx7sZ4bEFoJCy4YM/edit#).
+ * [Meeting Calendar](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MGRnMGJzdjhmY3E2aDhhOTZlOXV0M211dXBfMjAyMzAyMDdUMDIwMDAwWiBodWFuZzY4OTRAbQ)
 
 ## Contact
-- Slack: [#sig-robotics]()
+
+- [#sig-Robotics slack channel](https://join.slack.com/t/kubeedge/shared_invite/enQtNjc0MTg2NTg2MTk0LWJmOTBmOGRkZWNhMTVkNGU1ZjkwNDY4MTY4YTAwNDAyMjRkMjdlMjIzYmMxODY1NGZjYzc4MWM5YmIxZjU1ZDI)
```
Comment (fujitatomoya) [inline sig-robotics/README.md:16]: Do we actually have this dedicated channel named `sig-Robotics`? i cannot find KubeEdge slack...

Comment (kevin-wangzefeng) [inline sig-robotics/README.md:16]:
```suggestion
- [#sig-Robotics slack channel](https://kubeedge.slack.com/archives/C05H44JKB0Q)
```

**Comment by kevin-wangzefeng** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**